utils/R/convert.input.R

```diff
@@ -5,228 +5,245 @@
+      } else if (formatname == "Sipnet.climna") {
+        outputtype <- "clim"
+      } else if (formatname == "DALEC meteorology") {
+        outputtype <- "dalec.dat"
+      } else if (formatname == "LINKAGES met") {
+        outputtype <- "linkages.dat"
+      } else {
+        logger.severe(paste("Unknown formatname", formatname))
       }
     }
     
     # create curl options
     if (!is.null(browndog$username) && !is.null(browndog$password)) {
-      userpwd <- paste(browndog$username, browndog$password, sep=":")
+      userpwd <- paste(browndog$username, browndog$password, sep = ":")
       curloptions <- list(userpwd = userpwd, httpauth = 1L)
     }
-    curloptions <- c(curloptions, followlocation=TRUE)
+    curloptions <- c(curloptions, followlocation = TRUE)
     
-    # check if we can do conversion 
-    out.html <- getURL(paste0(
-      "http://dap-dev.ncsa.illinois.edu:8184/inputs/", browndog$inputtype), .opts = curloptions)
-    if(outputtype %in% unlist(strsplit(out.html, '\n'))){
-      logger.info(paste("Conversion from", browndog$inputtype, "to", outputtype, "through Brown Dog"))
+    # check if we can do conversion
+    out.html <- getURL(paste0("http://dap-dev.ncsa.illinois.edu:8184/inputs/", 
+                              browndog$inputtype), .opts = curloptions)
+    if (outputtype %in% unlist(strsplit(out.html, "\n"))) {
+      logger.info(paste("Conversion from", browndog$inputtype, "to", outputtype, 
+                        "through Brown Dog"))
       conversion <- "browndog"
     }
   }
   
-  if(conversion == "browndog"){
-    url <- file.path(browndog$url,outputtype) 
+  if (conversion == "browndog") {
+    url <- file.path(browndog$url, outputtype)
     
-    # loop over files in localhost and zip to send to Brown Dog 
-    files <- list.files(dbfile$file_path, pattern=dbfile$file_name)
-    files <- grep(dbfile$file_name, files,value=T)
-    zipfile <- paste0(dbfile$file_name,".", browndog$inputtype)
-    system(paste("cd", dbfile$file_path, "; zip", zipfile,  paste(files, collapse = " ")))
-    zipfile <- file.path(dbfile$file_path, zipfile) 
+    # loop over files in localhost and zip to send to Brown Dog
+    files <- list.files(dbfile$file_path, pattern = dbfile$file_name)
+    files <- grep(dbfile$file_name, files, value = TRUE)
+    zipfile <- paste0(dbfile$file_name, ".", browndog$inputtype)
+    system(paste("cd", dbfile$file_path, "; zip", zipfile, paste(files, collapse = " ")))
+    zipfile <- file.path(dbfile$file_path, zipfile)
     
     # check for and create output folder
-    if(!file.exists(outfolder)){
-      dir.create(outfolder, showWarnings=FALSE, recursive=TRUE)
+    if (!file.exists(outfolder)) {
+      dir.create(outfolder, showWarnings = FALSE, recursive = TRUE)
     }
     
     # post zipped file to Brown Dog
-    html <- postForm(url,"fileData" = fileUpload(zipfile), .opts = curloptions)
+    html <- postForm(url, fileData = fileUpload(zipfile), .opts = curloptions)
```

Comment:
@robkooper was there a reason fileData was quoted here that's BrownDog specific or is Ben's fix correct?